src/components/Channel/Channel.tsx

```diff
@@ -708,7 +708,10 @@ const ChannelWithContext = <
         ) {
           const mostRecentMessage =
             channel.state.messages[channel.state.messages.length - 1];
-          await queryAfterMessage(mostRecentMessage.id, 5);
+          await queryAfterMessage(
+            mostRecentMessage.id,
+            10 - channel.state.messages.length,
```

Comment:
this could become negative?

Reply:
Nope, it's wrapped under `if` condition!! All of this complex logic is mainly for android-the-wierdo